test/cases/api.sh

```diff
@@ -90,6 +90,11 @@ cat > "$REQUEST_FILE" << EOF
       "architecture": "$ARCH",
       "image_type": "ami",
       "repositories": $(jq ".\"$ARCH\"" /usr/share/tests/osbuild-composer/repositories/"$DISTRO".json),
+      "customizations": {
+        "packages": [
+          "postgres",
```

Comment:
Hm I guess there's no way to verify this if it actually gets installed since we don't boot the image. Maybe something to do in future, we would check the subscriptions that way as well as those are currently untested in the same way.

Reply:
Yeah, was thinking the same. We definitely should boot the image, and check all these things. I guess for a follow-up :)

Reply:
It seems using  [`aws ec2-instance-connect send-ssh-public-key`](https://docs.aws.amazon.com/cli/latest/reference/ec2-instance-connect/send-ssh-public-key.html) we could inject a public ssh key after creating the instance. Doesn't need to be a part of this PR imo but worth doing in future I think.